life_server/Functions/WantedSystem/fn_wantedAdd.sqf

```diff
@@ -8,88 +8,50 @@
     Description:
     Adds or appends a unit to the wanted list.
 */
-private ["_uid","_type","_index","_data","_crimes","_val","_customBounty","_name","_pastCrimes","_query","_queryResult"];
-_uid = [_this,0,"",[""]] call BIS_fnc_param;
-_name = [_this,1,"",[""]] call BIS_fnc_param;
-_type = [_this,2,"",[""]] call BIS_fnc_param;
-_customBounty = [_this,3,-1,[0]] call BIS_fnc_param;
-if (_uid isEqualTo "" || _type isEqualTo "" || _name isEqualTo "") exitWith {}; //Bad data passed.
+params [
+    ["_uid","",[""]],
+    ["_name","",[""]],
+    ["_type","",[""]],
+    ["_customBounty",-1,[0]]
+];
+
+if (_uid isEqualTo "" || {_type isEqualTo ""} || {_name isEqualTo ""}) exitWith {}; //Bad data passed.
 
 //What is the crime?
-switch (_type) do
-{
-    case "187V": {_type = ["187V",650]};
-    case "187": {_type = ["187",2000]};
-    case "901": {_type = ["901",450]};
-    case "215": {_type = ["215",200]};
-    case "213": {_type = ["213",1000]};
-    case "211": {_type = ["211",100]};
-    case "207": {_type = ["207",350]};
-    case "207A": {_type = ["207A",200]};
-    case "390": {_type = ["390",1500]};
-    case "487": {_type = ["487",150]};
-    case "488": {_type = ["488",70]};
-    case "480": {_type = ["480",100]};
-    case "481": {_type = ["481",100]};
-    case "482": {_type = ["482",500]};
-    case "483": {_type = ["483",950]};
-    case "459": {_type = ["459",650]};
-    case "666": {_type = ["666",200]};
-    case "667": {_type = ["667",4500]};
-    case "668": {_type = ["668",1500]};
+private _crimesConfig = getArray(missionConfigFile >> "Life_Settings" >> "crimes");
```

Comment:
@fresqo No. That'd reference the macro. This is using the class Life_Settings. This is essentially the same though...

Life_Settings: [See Here](https://github.com/AsYetUntitled/Framework/blob/3f9a27418ff307c300a11b89b2e7c566f4ca2b7e/Altis_Life.Altis/config/Config_Master.hpp#L7)
LIFE_SETTINGS: [See Here](https://github.com/AsYetUntitled/Framework/blob/3f9a27418ff307c300a11b89b2e7c566f4ca2b7e/Altis_Life.Altis/initPlayerServer.sqf#L1)